analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -101,6 +101,18 @@ import org.ossreviewtoolkit.utils.spdx.SpdxOperator
 
 fun Artifact.identifier() = "$groupId:$artifactId:$version"
 
+/**
+ * Return the path to this file or a corresponding message if the file is unknown.
+ */
+private val File?.safePath: String
+    get() = this?.invariantSeparatorsPath ?: "<unknown file>"
+
+/**
+ * Return the path to the pom file affected by this exception handling an undefined file gracefully.
+ */
+private val ProjectBuildingException.pomPath: String
+    get() = pomFile.safePath
```

Comment:
I believe this helper function is a bit overkill, though, and I'd simply use `e.pomFile.safePath` below for clarity that the path is *safe*.